testing/apps/reviews/reviews.go

```diff
@@ -75,6 +75,16 @@ func main() {
 
 	http.HandleFunc("/reviews", reviewsHandler)
 	http.HandleFunc("/health", func(w http.ResponseWriter, r *http.Request) {
+		conf := struct {
+			EnableRatings bool
+			StarColor     string
+		}{
+			EnableRatings: enableRatings,
+			StarColor:     starColor,
+		}
+
+		data, _ := json.Marshal(&conf)
+		w.Write(data)
 		w.WriteHeader(http.StatusOK)
```

Comment:
This was added to help with debugging proxying issues. It can be moved to a separate PR if you think that would be more appropriate --but it is handy to have it around.